File: 1._Belajar_Python/1._Latihan_Variasi/Latihan_Input_menentukan_jenis_Segitiga.py
# Program menentukan jenis segitiga 

print ("\n Program menentukan jenis segitiga \n")


a = int (input ("Sisi 1 :"))
b = int (input ("Sisi 2 :"))
c = int (input ("Sisi 3 :"))


if a + b > c and a + c > b and b + c > a:
    
    if a == b == c:
        print ("Segitiga Sama Sisi")
        
    elif a == b or b == c or a == c:
        print ("Segitiga Sama Kaki")
            
    else:
        print ("Segitiga sembarang")
                
else:
    print ("Bukan segitiga")

File: 1._Belajar_JavaScript_Console/1._Latihan_Variasi/Latihan_Input_menentukan_jenis_Segitiga.js
// Program menentukan jenis segitiga 

console.log ("\n Program menentukan jenis segitiga \n")


var a = Number (prompt ("Sisi 1 : "))
var b = Number (prompt ("Sisi 2 : "))
var c = Number (prompt ("Sisi 3 : "))


if ((a + b > c) && (a + c > b) && (b + c > a)) {
    
    if ((a == b) && (b == c)) {
        console.log ("Segitiga sama sisi")
    }
    
    else if ((a == b) || (b == c) || (a == c)) {
        console.log ("Segitiga sama kaki")
    }
    
    else {
        console.log ("Segitiga Sembarang")
    }
}

else {
    console.log ("Bukan segitiga")
}